internal/services/logic/logic_app_standard_resource.go

```diff
@@ -1024,32 +1056,38 @@ func schemaLogicAppStandardIpRestriction() *pluginsdk.Schema {
 	}
 }
 
-func flattenLogicAppStandardAppSettings(input map[string]*string) map[string]string {
+func flattenLogicAppStandardAppSettings(input *map[string]string) map[string]string {
```

Comment:
as per the data source, can probably be removed and replaced with `pointer.From()`.